map/src/main/java/openlr/map/utils/BearingDestinationRouting.java

```diff
@@ -0,0 +1,130 @@
+package openlr.map.utils;
+
+import openlr.map.GeoCoordinates;
+import openlr.map.Line;
+
+
+import java.util.Iterator;
+
+/**
+ * @since 1.4.4
+ * Find the point(Bearing destination) along the road network which is used to estimate the bearing of the LRP
+ */
+public class BearingDestinationRouting {
```

Comment:
I'm not sure the name `BearingDestinationRouting` gives the best picture of the purpose of the class.  It's not really routing.  I think a better name would be `BearingPointCalculator`.

Reply:
482ec5f58091993383b0ec513ceafb5223fa03d7